Source/Eto/Forms/Controls/Control.cs

```diff
@@ -50,6 +50,14 @@ public object Tag
 
 		static readonly object TagKey = new object();
 
+		static readonly object ResourcesKey = new object();
+
+		public Dictionary<object, object> Resources
+		{
+			get { return Properties.Create<Dictionary<object, object>>(ResourcesKey); }
+			set { Properties.Set(ResourcesKey, value); }
+		}
+
```

Comment:
This was added to be more 'compatible' with WPF or other xaml-based frameworks.. We already have 'properties', so is this really needed?